app/services/courses_service.rb

```diff
@@ -0,0 +1,24 @@
+class CoursesService
+  def self.create_guest_membership(course, user)
+    membership = UserCourseMembership.new { |m|
+      m.role = 3
```

Comment:
Please document all constants in an appropriate place and cross-reference it here.